guides/Usage.md

```diff
@@ -6,14 +6,14 @@ Sourcery is a command line tool, you can either run it manually or in a custom b
 $ ./sourcery --sources <sources path> --templates <templates path> --output <output path>
 ```
 
-> Note: this command may be different depending on the way in which you installed Sourcery (see [Installing](installing.html))
+> Note: this command may be different depending on the way in which you installed Sourcery (see [Installing](Installing.md))
```

Comment:
I believe this and other links in md files are not broken. They point to online documentation which is in html files. md files in guides folder are used to generate html documentation with jazzy. You can use it to generate documentation locally so that all links work.

Reply:
Oh, I see.
Should I close this PR them?